llama_index/indices/vector_store/base.py

```diff
@@ -36,19 +45,31 @@ class VectorStoreIndex(BaseIndex[IndexDict]):
     def __init__(
         self,
         nodes: Optional[Sequence[BaseNode]] = None,
-        objects: Optional[Sequence[IndexNode]] = None,
-        index_struct: Optional[IndexDict] = None,
-        service_context: Optional[ServiceContext] = None,
-        storage_context: Optional[StorageContext] = None,
+        # vector store index params
         use_async: bool = False,
         store_nodes_override: bool = False,
+        embed_model: Optional[EmbedType] = None,
         insert_batch_size: int = 2048,
```

Comment:
if i want to customize chunk size, do i define the explicit transformation with the specific chunk size? 

(this is fine, but it just means `from_documents` will be one extra line of code to specify the transformation`

Reply:
You can set `Settings.chunk_size`, `Settings.node_parser/text_splitter`, or pass in the transformation yes